account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -0,0 +1,342 @@
+        comodel_name='account.tax.template',
+        string='Default Sales Tax',
+    )
+    force_sale_tax = fields.Boolean(
+        string='Force Sale Tax In All Products',
+        help='Set default sales tax to all products.\n'
+             'If smart search product tax is also enabled matches founded '
+             'will overwrite default taxes, but not founded will remain',
+    )
+    default_purchase_tax_id = fields.Many2one(
+        comodel_name='account.tax.template',
+        string='Default Purchase Tax',
+    )
+    force_purchase_tax = fields.Boolean(
+        string='Force Purchase Tax In All Products',
+        help='Set default purchase tax to all products.\n'
+             'If smart search product tax is also enabled matches founded '
+             'will overwrite default taxes, but not founded will remain',
+    )
+    # ACCOUNTS
+    smart_search_specific_account = fields.Boolean(
+        default=True,
+        help='Go over specific accounts in actual company to match and set '
+             'equivalent taxes in the new company.\n'
+             'Applies to products, categories, partners, ...',
+    )
+    smart_search_fiscal_position = fields.Boolean(
+        default=True,
+        help='Go over partner fiscal positions in actual company to match '
+             'and set equivalent fiscal positions in the new company.',
+    )
+    update_default_accounts = fields.Boolean(
+        help='Update default accounts defined in account chart template',
+    )
+    account_receivable_id = fields.Many2one(
+        comodel_name='account.account.template',
+        string='Default Receivable Account',
+    )
+    account_payable_id = fields.Many2one(
+        comodel_name='account.account.template',
+        string='Default Payable Account',
+    )
+    account_income_categ_id = fields.Many2one(
+        comodel_name='account.account.template',
+        string='Default Category Income Account',
+    )
+    account_expense_categ_id = fields.Many2one(
+        comodel_name='account.account.template',
+        string='Default Category Expense Account',
+    )
+
+    def install_chart_account(self):
+        """ install a chart of accounts for the given company """
+        user_company = self.env.user.company_id
+        self.env.user.company_id = self.new_company_id
+        self.sudo().chart_template_id.try_loading_for_current_company()
+        self.env.user.company_id = user_company
+
+    def create_bank_journals(self):
+        AccountJournal = self.env['account.journal'].sudo()
+        bank_journals = AccountJournal.search([
+            ('type', '=', 'bank'),
+            ('company_id', '=', self.new_company_id.id),
+        ])
+        vals = {
+            'type': 'bank',
+            'company_id': self.new_company_id.id,
+        }
+        for i, bank in enumerate(self.bank_ids):
+            vals.update({
+                'name': bank.acc_number,
+                'bank_acc_number': bank.acc_number,
```

Comment:
```suggestion
                'bank_acc_number': bank.acc_number,
                'partner_id': self.new_company_id.partner_id.id,
```

Reply:
This is creating the account.journal, not the res.partner.bank record, so not needed. It's handled automatically by Odoo code.